joplin/api/schema.py

```diff
@@ -421,41 +449,9 @@ class Language(graphene.Enum):
     BURMESE = 'my'
 
 
-class ServicePageStepLocationBlock(graphene.ObjectType):
```

Comment:
again, wish we could use this pattern to create queryable things instead of just hardcoding our location page data in. Maybe we should make an issue for queryable streamfields and consider grapple as a possible way to make that happen

Reply:
I agree!

Reply:
could you make an issue and link it?

Reply:
https://github.com/cityofaustin/techstack/issues/3953